lib/alert.py

```diff
@@ -28,11 +29,15 @@ def approved_at_date(self):
 
     @property
     def expires_date(self):
-        return self.cancelled_at_date
+        return self.cancelled_at_date or self.finishes_at_date
+
+    @property
+    def finishes_at_date(self):
+        return AlertDate(self.finishes_at)
 
     @property
     def cancelled_at_date(self):
-        return AlertDate(self.cancelled_at)
+        return self.cancelled_at and AlertDate(self.cancelled_at)
```

Comment:
I’d prefer this return `None` rather than `False` if there isn’t a value present.

Reply:
I agree that's more accurate. I've amended that commit to return `None` (by default).